upf/upf_cli.c

```diff
@@ -728,6 +730,8 @@ upf_gtpu_endpoint_add_del_command_fn (vlib_main_t * vm,
   int rv;
   u8 *s;
 
+  upf_proxy_init (vm);
```

Comment:
why does this cli command the proxy to be ready? I don't see that is has a relation to the proxy process.
And if it does, then the call need to go into vnet_upf_upip_add_del.

Reply:
Fixed proxy init, now this is no longer needed and thus removed